protocol-contracts-solana/tests/universal_nft.ts

```
@@ -0,0 +1,269 @@
+          .rpc();
+        
+        console.log("✅ Burn transaction:", tx);
+        
+        // Verify the NFT was burned
+        const tokenAccount = await provider.connection.getTokenAccountBalance(recipientTokenAccount);
+        expect(tokenAccount.value.amount).toBe("0");
+        
+        // Verify replay marker was created
+        const replayMarkerAccount = await provider.connection.getAccountInfo(replayMarkerPda);
+        expect(replayMarkerAccount).toBeTruthy();
+        
```

Biome (2.1.2)

[error] This code will never be reached ...

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)